src/karmen_frontend/src/components/forms/form-utils.js

```diff
@@ -1,7 +1,9 @@
 import React from "react";
 
-export const FormInputs = ({ definition, updateValue }) => {
-  const optionRows = Object.keys(definition).map((name) => {
+import Collapsible from "../utils/collapsible";
+
+const InputSwitch = ({ definition, updateValue }) => {
```

Comment:
`InputSwitch` is not very self-explanatory. Looking at the code, it simply looks like any kind of form input. Can we please improve the naming?

Reply:
Generally speaking, I'm not a fan of self-made form solution. But it's probably too late to switch to something battle-proven like [Formik](https://jaredpalmer.com/formik/docs/overview).

Reply:
From my point of view it is a Switch that returns html inputs, so the naming should be ok, but you or @tibor-kulcar can suggest a better naming. Regarding to self-made solution: Yes! agree, and I have spoke about it above.

Reply:
I suggest something like `FormField`.

Reply:
It returns a set of fields, so it would be a FormFields if I get you suggestion.

Reply:
OK

Reply:
Done.